jkube-kit/doc/src/main/asciidoc/inc/build/_openshift-build.adoc

```diff
@@ -1,47 +1,3 @@
-[[jkubeBuild]]
-=== *{task-prefix}Build*
-
-This task is for building container images for your application.
-
-ifeval::["{task-prefix}" == "k8s"]
-[[build-kubernetes]]
-==== Kubernetes Build
-
-A normal Docker build is performed by default.For Kubernetes builds the {plugin} uses the Docker remote API so the URL of your Docker Daemon must be specified.The URL can be specified by the dockerHost or machine configuration, or by the `DOCKER_HOST` environment variable.
-
-The Docker remote API supports communication via SSL and authentication with certificates.The path to the certificates can be specified by the certPath or machine configuration, or by the `DOCKER_CERT_PATH` environment variable.
-
-If you don't have access to docker daemon, you can change build strategy using `buildStrategy` option in Groovy configuration like this:
-
-.Example of overriding default build strategy
-[source,groovy,subs="attributes+"]
-----
-{pluginExtension} {
-    buildStrategy = 'jib'
-}
-----
-
-These are the different options supported by `buildStrategy`:
-
-.JKube Build Strategies
-[cols="1,5"]
-|===
-| *buildStrategy* | *Description*
-| `docker`
-| Docker build with a binary source
-
-| `jib`
-| Deamonless container image creation using https://github.com/GoogleContainerTools/jib[JIB] build
-|===
-
-{plugin} by default tries to build up an opinionated <<config-image, Image Configuration>> by inspecting `build.gradle`.
-You can also provide your own <<dockerfile-scenario, Dockerfile>> or provide Custom ImageConfiguration via <<groovy-scenario-image, Groovy DSL configuration>>.
-endif::[]
-
-ifeval::["{task-prefix}" == "oc"]
-
-[[build-openshift]]
```

Comment:
this links seems to be lost

Reply:
I removed this heading on purpose as it seemed a bit weird to have a section of Kubernetes Build/ OpenShift Build in [`oc:build` section](https://eclipse.dev/jkube/docs/openshift-maven-plugin/#jkube:build)

![Screenshot_20240208_155940](https://github.com/eclipse/jkube/assets/13834498/8849e17a-f3fe-4494-8988-b0452155cbf4)

Reply:
I'm talking about the link, not the header. That link is referenced in the documentation